File: pano_progra/practica_2.py
mes = 1
dinero = []
while mes <= 12:
  ahorro = float(input(f"Ingresa el dinero que ahorraras en el mes {mes}: "))
  if ahorro <= 0:
    print("Invalido")
    continue
  dinero.append(ahorro)
  mes += 1

print(f"Esto es el total que obtendras al final del año: {sum(dinero)}")

File: pano_progra/practica_2.cs
using System;
using System.Collections.Generic;
using System.Linq;

class Program
{
    static void Main(string[] args)
    {
        int mes = 1;
        List<double> dinero = new List<double>();

        while (mes <= 12)
        {
            Console.Write($"Ingresa el dinero que ahorraras en el mes {mes}: ");
            double ahorro = Convert.ToDouble(Console.ReadLine());

            if (ahorro <= 0)
            {
                Console.WriteLine("Invalido");
                continue;
            }

            dinero.Add(ahorro);
            mes++;
        }

        Console.WriteLine($"Esto es el total que obtendras al final del año: {dinero.Sum()}");
    }
}
